datastore/src/main/java/com/google/datastore/snippets/TaskList.java

```diff
@@ -70,14 +70,18 @@ Key addTask(String description) {
    * Marks a task entity as done.
    *
    * @param id The ID of the task entity as given by {@link Key#id()}
-   * @throws DatastoreException if the task does not exist
+   * @return true if the task was found, false if not
+   * @throws DatastoreException if the transaction commit fails
```

Comment:
For similar reason to the above (there is also get) if you want to be picky you can say "if Datastore transaction fails"